build_scripts/windows/scripts/build.cmd

```diff
@@ -1,5 +1,9 @@
 @echo off
 SetLocal EnableDelayedExpansion
+
+REM Double colon :: should not be used in parentheses blocks, so we use REM.
+REM See https://stackoverflow.com/a/12407934/2199657
```

Comment:
- https://stackoverflow.com/questions/16632524/what-does-double-colon-mean-in-dos-batch-files
- https://stackoverflow.com/questions/3978655/windows-batch-file-with-goto-command-not-working
- https://stackoverflow.com/questions/12407800/which-comment-style-should-i-use-in-batch-files